core/src/main/java/com/nuodb/migrator/backup/loader/LoadTableForkWork.java

```diff
@@ -112,6 +114,28 @@ public void execute() throws Exception {
                 backupLoaderManager.afterLoadRow(this, loadTable, row);
             }
             commitExecutor.finish();
+            // added to alter the table after loading the data
+            try {
+                // added gen always fix
+                GlobalStore global = GlobalStore.getInstance();
+                CommitStrategy commitStrategy = backupLoaderContext.getCommitStrategy() != null
+                        ? backupLoaderContext.getCommitStrategy()
+                        : new BatchCommitStrategy();
+
+                String alterScript = global.alterScript(loadTable);
+                if (alterScript != null) {
+                    PreparedStatement newStatement = getSession().getConnection().prepareStatement(alterScript);
+                    commitAlter = commitStrategy.createCommitExecutor(newStatement, null);
+
+                    commitAlter.execute();
+                    commitAlter.finish();
+                }
+            } catch (Exception e) {
+                if (logger.isDebugEnabled()) {
+                    logger.debug("Issue while executing Alter Statement after loading the data");
+                }
```

Comment:
Why debug? Normally we log exception as ERROR, WARN, or at least INFO, so that it shows up in the logs.

Also, can you either handle or exclude from the `catch`-clause `InterruptedException`? The interrupted status should be reset if we are suppressing `InterruptedException` or re-throwing it as something else.